src/olympia/activity/models.py

```diff
@@ -538,7 +538,7 @@ def to_string(self, type_=None):
                     validation = 'ignored'
 
                 file_ = self.f(u'<a href="{0}">{1}</a> (validation {2})',
-                               reverse('files.list', args=[arg.pk]),
+                               arg.get_url_path(arg.pk),
```

Comment:
are you sure about this?  File.get_url_path seems to be an instance method that doesn't require the id, and is the download link (rather than an admin manage page)

Reply:
I decided against the admin manage page because none of the other types in this method link to their admin manage page, so I will just keep it consistent. As for the parameter, it is required, but it's not the pk, but the `src` one. Updated in 100faa13abbdfe85f245b05d129107c015c5d629.